src/metabolic_ninja/resources.py

```diff
@@ -139,7 +144,7 @@ def retrieve_model_json(model_id, headers):
         response.raise_for_status()
         return response.json()
 
-    @marshal_with(PredictionJobSchema(many=True, exclude=("result",)))
+    @marshal_with(PredictionJobSchema(many=True, exclude=("result",)), 200)
```

Comment:
200 is not the default status? Or you just wanted to be explicit?

Reply:
The status code is required. Without that it will not be shown in the documentation and you get the warning `/usr/local/lib/python3.6/site-packages/apispec/core.py:111: UserWarning: Non-integer code not allowed in OpenAPI < 3`